qa/suites/crimson-rados-experimental/seastore/basic/clusters/fixed-1.yaml

```diff
@@ -10,5 +10,11 @@ overrides:
     conf:
       osd:
         osd shutdown pgref assert: true
+      osd.0:
+        crimson seastar cpu cores: 0-0
+      osd.1:
+        crimson seastar cpu cores: 1-1
+      osd.2:
+        crimson seastar cpu cores: 2-2
```

Comment:
Might be better to validate with multiple cores.

Reply:
Yeah, I agree.